src/legacy/core_plugins/kibana/public/visualize/listing/_listing.scss

```diff
@@ -2,9 +2,17 @@
 .visListingTable__typeIcon {
   margin-right: $euiSizeS;
   position: relative;
-  top: 2px;
+  top: -1px;
 }
 
 .visListingTable__typeImage {
   @include size($euiSize);
 }
+
+.visListingTable__experimentalIcon {
+  width: 24px;
+  vertical-align: baseline;
+  padding: 0 8px;    
+  margin-left: 8px;
```

Comment:
For all these `px` values can you use our EUI SASS variables instead: https://elastic.github.io/eui/#/guidelines/sass

Reply:
Since CI is quite unstable right now, I will merge this, and make a separate PR for the CSS :)

Reply:
kk